tests/integration/test_api_gateway.py

```diff
@@ -187,3 +189,49 @@ def test_api_gateway_lambda_proxy_integration():
     assert parsed_body.get('return_status_code') == 203
     assert parsed_body.get('return_headers') == {'foo': 'bar123'}
     assert parsed_body.get('pathParameters') == {'test_param1': 'foo1'}
+    result = requests.delete(url, data=json.dumps(data))
+    assert result.status_code == 404
+
+
+def test_api_gateway_lambda_proxy_integration_any_method():
+    # create lambda function
+    zip_file = testutil.create_lambda_archive(load_file(TEST_LAMBDA_PYTHON), get_content=True,
+        libs=TEST_LAMBDA_LIBS, runtime=LAMBDA_RUNTIME_PYTHON27)
+    testutil.create_lambda_function(func_name=TEST_LAMBDA_PROXY_BACKEND_ANY_METHOD,
+        zip_file=zip_file, runtime=LAMBDA_RUNTIME_PYTHON27)
+
+    # create API Gateway and connect it to the Lambda proxy backend
+    lambda_uri = aws_stack.lambda_function_arn(TEST_LAMBDA_PROXY_BACKEND_ANY_METHOD)
+    target_uri = 'arn:aws:apigateway:%s:lambda:path/2015-03-31/functions/%s/invocations' % (DEFAULT_REGION, lambda_uri)
```

Comment:
This uses the same URI syntax as in line 177 - can we extract this as a small util function `apigateway_invocations_arn(lambda_uri)` and add it to [`utils/aws/aws_stack.py`](https://github.com/localstack/localstack/blob/master/localstack/utils/aws/aws_stack.py#L222)?